test/tests/actions/debug_block/debug_print_actions_test.i

```diff
@@ -44,6 +44,7 @@
 []
 
 [Debug]
+  show_action_dependencies = true
```

Comment:
Please change the test specifications; we should actually have one test individually for each of these methods and should not run them all at once

Reply:
The `tests` actually contains the two exact specifications we want.

Reply:
And the same input is run twice to test the two specifications.